src/apis/study-rooms/get-study-rooms.ts

```diff
@@ -0,0 +1,26 @@
+import { http } from '../api';
+
+type StudyRoomsResponse = {
+  studyRooms: [
+    {
+      id: number;
+      name: string;
+      intro: string;
+      step: string;
+      participantCount: number;
+    },
+  ];
+  pageDto: {
+    current: number;
+    total: number;
+  };
+};
+
+type StudyRoomsData = {
+  data: { page: number };
+};
+
+export const getStudyRooms = async (data: StudyRoomsData): Promise<StudyRoomsResponse> => {
+  const response = await http.get<StudyRoomsResponse>(`/studyrooms`, { ...data });
```

Comment:
단순 문자열만 표현한다면 백틱보다는 따옴표가 좋을 것 같아요~
```suggestion
  const response = await http.get<StudyRoomsResponse>('/studyrooms', { ...data });
```

Reply:
get 메소드의 2번째 인수로 `{ ...data }` 이렇게 넘겨주는게 맞나요?